### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Gsantomaggio**

### CODE REVIEW

File: Tests/IntegrationTest.cs [hunk 1]
```diff
@@ -16,34 +16,32 @@ namespace Tests;
...
+    protected readonly TimeSpan WaitSpan = TimeSpan.FromSeconds(5);
 
-    protected IConnection? _connection;
-    protected IManagement? _management;
-    protected string _queueName;
-    protected string _exchangeName;
-    protected string _containerId = $"integration-test-{Now}";
+    protected IConnection? Connection;
+    protected IManagement? Management;
+    protected string QueueName;
```
Comment (lukebakken) [inline Tests/IntegrationTest.cs:24]: Could you please change your IDE to not make this change? In the other .NET clients the style we use for fields is to prefix them with an underscore.

**Review by lukebakken — CHANGES_REQUESTED** [review]
Formatting request.

File: Tests/IntegrationTest.cs (hunk 1 above)
Comment (lukebakken) [reply Tests/IntegrationTest.cs:24]: Let me check the `.editorconfig` to make sure this is being enforced.

Comment (Gsantomaggio) [reply Tests/IntegrationTest.cs:24]: Should be the `_` only for private member? btw I will change ! :)

Comment (Gsantomaggio) [reply Tests/IntegrationTest.cs:24]: >Let me check the .editorconfig to make sure this is being enforced. 

Ok thank you

Comment (lukebakken) [reply Tests/IntegrationTest.cs:24]: @Gsantomaggio I added enforcement of this rule via 005e072d4d192d23c6ec03317334ed31e45179d8. I'll make sure the other .NET clients do the same. Thanks.

**Pull request merged**